stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.21.0
+version: 1.21.1
```

Comment:
A change got in before yours. You will need to rebase on master and update this to `1.22.1`